modules/layers/src/text-layer/text-layer.js

```diff
@@ -311,10 +348,11 @@ export default class TextLayer extends CompositeLayer {
       }),
       {
         data,
-
         getIcon: d => d.text,
+        getLineLength: d => this.getLineLength(d),
```

Comment:
Either `d => d.lineLengths[d.lineIndex]` or `this.getLineLength`. Why wrap functions that already work?

I personally prefer defining these accessors inline. It's a lot easier to read.